README.md

```diff
@@ -23,6 +26,37 @@ This repository contains an SDK and Jupyter notebooks with examples to showcase
 - [Build Status](#BuildStatus)
 - [Additional References](#Refs)
 
+# <a name="intro"></a> 
+
+# Overview of Interpret-Community
+Interpret-community is a practical and community-driven repository that hosts a wide range of community-developed machine learning interpretability techniques. This repository makes it easy for anyone involved in the development of an AI system to achieve transparency around their machine learning models. Data scientists, machine learning engineers, and researchers can easily add their own interpretability techniques via the set of extension hooks built into the parent repository, Interpret, and expand this repository to include their custom-made interpretability techniques.
+
+
+This repo incorporates community-driven state-of-the-art interpretability solutions under one roof with a unified set of data structures and API. Users could experiment with different interpretability methods, and/or add their custom-made interpretability techniques and perform comparative analysis to evaluate their brand new explainers. Using these tools, one can explain machine learning models globally on all data, or locally on a specific data point using the state-of-art technologies in an easy-to-use and scalable fashion. In particular, this released open source toolkit: 

```

Comment:
nit: maybe remove state-of-the-art at the start or end of the paragraph (it's mentioned twice in the same paragraph so seems redundant)

Reply:
I see state-of-the-art mentioned 6 times in the doc, maybe we can use different wording for some of the cases

Reply:
suggest change solutions (we don't solve yet) to techniques or methods 
change perform to more easily perform
Suggest we pick either techniques or methods or tools as word so we are consistent.